pkg/client/connector/install.go

```diff
@@ -1029,6 +1035,30 @@ func (ki *installer) managerDeployment(env client.Env) *kates.Deployment {
 	if env.SystemAPort != "" {
 		containerEnv = append(containerEnv, corev1.EnvVar{Name: "SYSTEMA_PORT", Value: env.SystemAPort})
 	}
+	// If addLicense is true, we mount the secret as a volume into the traffic-manager
+	// and then we mount that volume to a path in the container that the traffic-manager
+	// knows about and can read from.
+	var licenseVolume []corev1.Volume
+	var licenseVolumeMount []corev1.VolumeMount
+	if addLicense {
+		licenseVolume = []corev1.Volume{
+			{
+				Name: "license",
+				VolumeSource: corev1.VolumeSource{
+					Secret: &corev1.SecretVolumeSource{
+						SecretName: managerLicenseName,
+					},
+				},
+			},
+		}
+		licenseVolumeMount = []corev1.VolumeMount{
+			{
+				Name:      "license",
+				ReadOnly:  true,
+				MountPath: "/home/telepresence/",
+			},
+		}
+	}
```

Comment:
Now wait a minute, if we're going to get it as a volume from a secret, wouldn't it be way simpler to have it get mounted directly in the agent workloads? Instead of mounting it in the manager Pod and passing it to the agents over gRPC?

Reply:
Oh, probably because RBAC, right?

Reply:
Yeah since I was using secrets, it'd be an RBAC escalation and normal users would need the ability to read secrets which I think some organizations are rightly opposed to allowing their developers to do.  

Additionally secrets are namespace bound so then we'd have to put that secret in every namespace intercepted workloads were in, which I thought might be a headache.  

All that being said, secrets might not be the best™️ way to do this forever, so I will revisit that in the next iteration of license work.